packages/legend-query-builder/src/stores/shared/ValueSpecificationEditorHelper.ts

```diff
@@ -125,6 +137,88 @@ const createMockPrimitiveValueSpecification = (
   );
 };
 
+export const relationalDataTypeToPrimitiveType = (
```

Comment:
lets avoid duplicate cod!!
`getPrimitiveTypeFromRelationalType``

Reply:
i didn't realize that this function exists..